### PR TITLE
Decreases the accuracy of lasergun turrets

### PR DESCRIPTION
**Pull request by bowlofcereal**
## About The Pull Request

C̶u̶r̶r̶e̶n̶t̶l̶y̶,̶ ̶l̶a̶s̶e̶r̶ ̶g̶u̶n̶ ̶t̶u̶r̶r̶e̶t̶s̶ ̶w̶i̶t̶h̶ ̶h̶i̶t̶s̶c̶a̶n̶ ̶a̶r̶e̶ ̶a̶b̶l̶e̶ ̶t̶o̶ ̶h̶i̶t̶ ̶x̶e̶n̶o̶s̶ ̶w̶i̶t̶h̶ ̶n̶e̶x̶t̶ ̶t̶o̶ ̶n̶o̶ ̶c̶h̶a̶n̶c̶e̶ ̶o̶f̶ ̶m̶i̶s̶s̶i̶n̶g̶ ̶t̶h̶e̶ ̶t̶a̶r̶g̶e̶t̶.̶ ̶I̶'̶v̶e̶ ̶s̶e̶e̶n̶ ̶c̶o̶u̶n̶t̶l̶e̶s̶s̶ ̶x̶e̶n̶o̶s̶ ̶g̶e̶t̶ ̶k̶i̶l̶l̶e̶d̶ ̶a̶l̶m̶o̶s̶t̶ ̶i̶n̶s̶t̶a̶n̶t̶a̶n̶e̶o̶u̶s̶l̶y̶ ̶b̶y̶ ̶l̶a̶s̶e̶r̶ ̶g̶u̶n̶ ̶m̶o̶u̶n̶t̶e̶d̶ ̶o̶v̶e̶r̶c̶h̶a̶r̶g̶e̶ ̶t̶u̶r̶r̶e̶t̶s̶.̶ ̶T̶h̶i̶s̶ ̶r̶e̶m̶o̶v̶e̶s̶ ̶t̶h̶e̶ ̶b̶u̶i̶l̶d̶-̶a̶-̶s̶e̶n̶t̶r̶y̶ ̶ ̶a̶t̶t̶a̶c̶h̶m̶e̶n̶t̶ ̶f̶r̶o̶m̶ ̶t̶h̶e̶ ̶l̶a̶s̶e̶r̶ ̶g̶u̶n̶ ̶w̶e̶a̶p̶o̶n̶s̶ ̶t̶o̶ ̶g̶e̶t̶ ̶r̶i̶d̶ ̶o̶f̶ ̶s̶o̶m̶e̶ ̶p̶o̶w̶e̶r̶c̶r̶e̶e̶p̶.̶

Decreases the accuracy of laser gun turrets instead of removing them as per maintainer request. (I spent 2h doing this help)

## Why It's Good For The Game

X̶e̶n̶o̶s̶ ̶a̶r̶e̶ ̶n̶o̶ ̶l̶o̶n̶g̶e̶r̶ ̶i̶n̶s̶t̶a̶n̶t̶l̶y̶ ̶k̶i̶l̶l̶e̶d̶ ̶b̶y̶ ̶l̶a̶s̶e̶r̶ ̶g̶u̶n̶ ̶t̶u̶r̶r̶e̶t̶ ̶c̶h̶e̶e̶s̶e̶ ̶a̶s̶ ̶i̶t̶ ̶n̶o̶ ̶l̶o̶n̶g̶e̶r̶ ̶i̶s̶ ̶p̶o̶s̶s̶i̶b̶l̶e̶
Less accurate hitscan laser turrets for less instantaneous death

## Changelog
:cl:
balance: r̶e̶m̶o̶v̶e̶s̶ ̶l̶a̶s̶e̶r̶g̶u̶n̶ ̶t̶u̶r̶r̶e̶t̶s̶ ̶f̶r̶o̶m̶ ̶b̶u̶i̶l̶d̶a̶s̶e̶n̶t̶r̶y̶ Lowers the accuracy of laser gun turrets
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
lol 👍 
Probably should look at other guns just incase for future reference.

**Comment by bowlofcereal** [comment]
> Probably should look at other guns just incase for future reference.

yeah im gonna test pretty much every build-a-sentry capable gun and either remove the attachment as a possibility or start changing the fuck out of values

**Comment by Lumipharon** [comment]
Thank you.
Even RR turrets are less awful.

**Comment by Kesserline** [comment]
Kinda a salt PR, knowing that it was because you died specifically during a round that you made that. I don't judge, but the fact that you kept exagerating the facts makes it a bit sad.

I spent the last round testing it with Jogoun, there are so many counters to those turrets, the mere fact that you can acid spit them, outscreen is the most obvious counter.

**Comment by bowlofcereal** [comment]
As per request from Bravemole, laser gun turrets have instead been given an accuracy penalty instead of being removed outright.

https://gyazo.com/12cc68fb72fa926eae6e8a8914d3a291

W̶e̶ ̶s̶p̶e̶n̶t̶ ̶2̶h̶ ̶d̶o̶i̶n̶g̶ ̶t̶h̶i̶s̶ ̶i̶n̶ ̶#̶c̶o̶d̶i̶n̶g̶ ̶h̶e̶l̶p̶ ̶m̶e̶

File: code/modules/projectiles/sentries.dm [hunk 1]
```diff
@@ -462,6 +466,14 @@
 		return nearby_target
 
 
+/obj/machinery/deployable/mounted/sentry/disassemble(mob/user)
+	. = ..()
+	var/obj/item/weapon/gun/gun = internal_item
+	if(CHECK_BITFIELD(gun.turret_flags, TURRET_INACCURATE))
+		gun.accuracy_mult /= 0.85
+		gun.scatter += 10
```
Comment (hyper2snyper) [inline code/modules/projectiles/sentries.dm:474]:
```suggestion
		gun.scatter -= 10
```
You are adding ten more here

File: code/modules/projectiles/sentries.dm [hunk 2]
```diff
@@ -54,6 +54,10 @@
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 
+	if(CHECK_BITFIELD(gun.turret_flags, TURRET_INACCURATE))
+		gun.accuracy_mult *= 0.85
```
Comment (BraveMole) [inline code/modules/projectiles/sentries.dm:58]:
```suggestion
		gun.accuracy_mult -= 0.15
```

File: code/modules/projectiles/sentries.dm [hunk 3]
```diff
@@ -462,6 +466,14 @@
 		return nearby_target
 
 
+/obj/machinery/deployable/mounted/sentry/disassemble(mob/user)
+	. = ..()
+	var/obj/item/weapon/gun/gun = internal_item
+	if(CHECK_BITFIELD(gun.turret_flags, TURRET_INACCURATE))
+		gun.accuracy_mult /= 0.85
```
Comment (BraveMole) [inline code/modules/projectiles/sentries.dm:473]:
```suggestion
		gun.accuracy_mult += 0.15
```

**Review by BraveMole — CHANGES_REQUESTED** [review]
0.15 and 10 (the value you are adding when deploying the turrets) should be define. That prevent people in the future changing the value when deploying, but forgetting from changing them when undeploying

**Review by TiviPlus — APPROVED** [review]
code ok pending david

**Review by David-Stormwell — APPROVED** [review]
Incase this wasn't clear enough in my first message, 👍 it should be fine.

**Pull request merged**